airgun/widgets.py

```diff
@@ -2378,3 +2422,33 @@ def fill(self, value):
 
     def read(self):
         return self.selected
+
+
+class SatPatternflyTable(BasePatternflyTable, Table):
```

Comment:
I think you should use `PatternflyTable` here instead of trying to include both.